src/api/mod.rs

```diff
@@ -132,6 +139,7 @@ impl CliCommands {
             CliCommands::Users(cmd) => cmd.run(global_options).await?,
             CliCommands::Upgrade(cmd) => cmd.run().await?,
             CliCommands::Config(cmd) => cmd.run(global_options).await?,
+            CliCommands::X509(cmd) => cmd.run(global_options).await?,
```

Comment:
Wondering if something more generic like `certificate` or `generate` with a `x509` subcommand would be better.

Reply:
Well we already have `device-certificates` and `ca-certificates` commands, I didn't put this under either of those because it's relevant to both (this cmd creates certificates that could be consumed by either of those command groups).

I did think about using `certificate` as the top level command but ended up not going that route because it felt like we had a pattern of "certificate" commands be commands that worked on already existing locally certificates being registered with the cloud; whereas this `x509` command explicitly doesn't talk to the cloud at all and is intended for local creation of certificates (likely to be consumed by subsequent `ca-certificates create` and `device-certificates-create` commands.

I also considered adding a single command that would create and register certificates with Peridio, but the complexity overload of representing that all in one command IMO did not seem worth it.

Definitely open to conversation here though, but will leave as-is unless a compelling point is raised.

Reply:
I'm also mirroring the `openssl x509` command https://docs.openssl.org/3.3/man1/openssl-x509/.